apm-agent-api/src/main/java/co/elastic/apm/api/Span.java

```diff
@@ -162,10 +194,10 @@
      *
      * @param key   The label key.
      * @param value The label value.
-     * @since 1.5.0, APM Server 6.7
+     * @since 1.9.0, APM Server 6.7
```

Comment:
[question] should it be `1.19` here for the `@since` javadoc tag instead ?

Reply:
Of course, good catch, thanks!